src/presentation/pages/Home/KnowHow/KnowHow.tsx

```diff
@@ -0,0 +1,48 @@
+import { Grid } from '@material-ui/core';
+import First from '../../../../app/assets/First.png';
+import Second from '../../../../app/assets/Second.png';
+import Third from '../../../../app/assets/Third.png';
```

Comment:
WDYT about adding the attribute `"baseUrl": "src"` at the `tsconfig.json` in order to prevent this kind of imports? they will look like this: 
```suggestion
import First from '/app/assets/First.png';
import Second from 'app/assets/Second.png';
import Third from '/app/assets/Third.png';
```

Reply:
great idea, thanks